src/fields/field_value_cache.cc

```diff
@@ -37,7 +37,8 @@ ElementCacheMap::ElementCacheMap()
 : elm_idx_(CacheMapElementNumber::get(), ElementCacheMap::undef_elem_idx),
   ready_to_reading_(false), element_eval_points_map_(nullptr), eval_point_data_(0),
   regions_starts_(2*ElementCacheMap::regions_in_chunk,ElementCacheMap::regions_in_chunk),
-  element_starts_(2*ElementCacheMap::elements_in_chunk,ElementCacheMap::elements_in_chunk) {}
+  element_starts_(2*ElementCacheMap::elements_in_chunk,ElementCacheMap::elements_in_chunk),
+  dh_(nullptr) {}
```

Comment:
Why we need DH here? It seems inappropriate since there may be multiple DH involved in single assembly.